src/mono/wasm/host/Options.cs

```diff
@@ -2068,7 +2070,7 @@ public override int Invoke(IEnumerable<string> arguments)
                 command.Options.WriteOptionDescriptions(CommandSet.Out);
                 return 0;
             }
-            return command.Invoke(new[] { "--help" });
+            return command.Invoke(s_help);
```

Comment:
Won't this be invoked only once?

Reply:
Not sure, at least not obviously looks invoked once.

Reply:
it's going to be invoked only when `--help` is passed, which we're assuming is response to human action or being launched or something else that doesn't happen much. suggest to revert changes in this file.

Reply:
Done